reports/Wisconsin.md

```diff
@@ -74,3 +74,15 @@ id: wi-milwaukee-2
 * [Location of tear gas, via Google Street View](https://www.google.com/maps/@43.0480627,-87.9185747,3a,75y,186.43h,95.71t/data=!3m6!1e1!3m4!1shjG96eAYHT1McDRHGI_3bA!2e0!7i16384!8i8192)
 * [Location of Reddit video, via Google Street View](https://www.google.com/maps/@43.0456964,-87.9185992,3a,75y,312.46h,104.94t/data=!3m6!1e1!3m4!1sK4uLnbJnDybp9dYDKowiww!2e0!7i16384!8i8192)
 
+
+### Police cut off protesters on Hoan Bridge | June 2nd
+
+Footage shows protesters peacefully marching across Hoan Bridge. Protesters are met by a line of police who force them back across the bridge. During this encounter, a scuffle breaks out - it is unclear what caused it. Police respond by shoving protesters and throwing smoke bombs. 2 protesters are arrested for unknown reasons. Police also fire less lethal munitions, hitting at least one person in the leg - seen in the second link. 
+
+tags: gas, protester, projectile, shoot, smoke-bomb, arrest
```

Comment:
I don't think we have a tag for smoke bomb. We can add it? But I feel like smoke isn't really like applicable for the brutality most of the time.